src/commands/QuotesCommand/QuotesCommand.js

```diff
@@ -0,0 +1,86 @@
+const BaseCommand = require('../../utils/structures/BaseCommand');
+const fs = require('fs');
+const { writeToFile } = require('./utils');
+
+const filePath = 'src/commands/QuotesCommand/quotes.json';
+
+module.exports = class QuotesCommand extends BaseCommand {
+    constructor() {
+        super('quote', 'textAnswers', []);
+
+        this.quotes = {};
+        fs.readFile(filePath, 'utf8', (err, contents) => {
+            if (err) {
+                console.error(err);
+                return;
+            }
+            this.quotes = JSON.parse(contents);
+            console.log(this.quotes);
+        });
+    }
+
+    async run(client, message, args) {
+        if (args[0]) {
+            if (this.quotes[args[0].toLowerCase()])
+                message.channel.send(this.quotes[args[0].toLowerCase()]);
+            else {
+                switch (args[0]) {
+                    case 'add':
```

Comment:
Maybe create a function to wrap each branch of the switch?
That way the main structure of the command is more readable